configs/sst_openstack.yaml

```diff
@@ -6,102 +6,330 @@ data:
+  - fence-agents-all
   - fence-virt
-  - libkcapi-fipscheck
-#  - gperftools
-#  - genisoimage
+  - findutils
+  - fio
+  - fontawesome-fonts
+  - fontpackages-filesystem
+  - galera
+  - gdisk
+  - genisoimage
+  - gettext
   - git
+  - git-core
+  - glibc
+  - glibc-langpack-en
+  - gmp
+  - grub2
+  - grub2-efi-x64
+  - grub2-efi-x64-modules
+  - gtk3
+  - gzip
   - haproxy
-  - hivex
-#  - hostname
-  - icu
+  - hdparm
+  - hostname
+  - httpd
+  - initscripts
   - ipa-client
   - ipmitool
+  - iproute
+  - iproute-tc
+  - ipset
+  - iptables
+  - iptables-ebtables
+  - iptables-libs
+  - iptables-services
+  - iputils
   - ipxe-bootimgs
-  - ipxe-roms
-  - ipxe-roms-qemu
-  - isns-utils
-  - jasper
-  - jbigkit
+  - iscsi-initiator-utils
+  - jansson
+  - jq
   - keepalived
-  - leveldb
-  - libknet1
-  - libnozzle1
-  - libknet1-plugins-all
-  - libcacard
-  - libcgroup
-  - liberation-fonts
-  - libev
-#  - libguestfs   # asamalik: it's a placeholder in another workload
-  - libiscsi
+  - kmod
+  - kpartx
+  - krb5-libs
+  - krb5-workstation
+  - libaio
+  - libatasmart
+  - libattr
+  - libblkid
+  - libcap
+  - libcgroup-tools
+  - libcom_err
+  - libcurl
+  - libgcc
+  - libgcrypt
+  - libgpg-error
+  - libguestfs
+  - libmemcached-libs
+  - libmicrohttpd
+  - libmnl
+  - libnotify
+  - libosinfo
+  - libpcap
+  - libpq
+  - libqb
   - librabbitmq
+  - librados2
+  - librdkafka
   - libreswan
-  - libwayland-client
-  - libwayland-egl
-  - libwayland-server
-  - libbasicobjects
-  - libcollection
-  - libdhash
-  - libini_config
-  - libpath_utils
-  - libref_array
+  - libseccomp
+  - libselinux-ruby
+  - libselinux-utils
+  - libstdc++
+  - libvirt-admin
+  - libvirt-client
+  - libvirt-daemon
+  - libvirt-daemon-config-nwfilter
+  - libvirt-daemon-driver-nodedev
+  - libvirt-daemon-driver-nwfilter
+  - libvirt-daemon-driver-qemu
+  - libvirt-daemon-driver-secret
+  - libvirt-daemon-driver-storage-core
+  - libvirt-libs
+  - libxml2
+  - lksctp-tools
+  - lldpad
+  - lm_sensors-libs
+  - logrotate
+  - lshw
+  - lsof
   - lsscsi
-  - mesa-filesystem
-  - mesa-dri-drivers
-  - mozjs78
-  - netcf
+  - lvm2
+  - lz4-libs
+  - mariadb
+  - mariadb-backup
+  - mariadb-connector-c
+  - mariadb-server-galera
+  - mariadb-server-utils
+  - mdadm
+  - memcached
+  - mod_auth_gssapi
+  - mod_auth_mellon
+  - mod_ssl
+  - ncurses
+  - ncurses-libs
+  - ndctl
+  - net-snmp
+  - net-snmp-agent-libs
+  - net-snmp-libs
+  - net-tools
+  - network-scripts
+  - nfs-utils
+  - nmap-ncat
+  - nodejs
+  - numactl
   - nvme-cli
   - nvmetcli
-  - oathtool
-  - OpenIPMI
-#  - openvswitch
-#  - ovn
+  - openblas-threads
+  - OpenIPMI-libs
+  - OpenIPMI-perl
+  - openldap
+  - openssh
+  - openssh-clients
+  - openssh-server
+  - openssl
+  - openssl-libs
+  - openssl-perl
   - pacemaker
+  - pacemaker-remote
+  - parted
+  - pciutils
+  - pcre
   - pcs
+  - perl-Carp
+  - perl-interpreter
+  - perl-libs
+  - pkgconf-pkg-config
+  - platform-python
```

Comment:
In the context of ELN, there is no platform-python.  We should probably exclude this one and just focus on the python3 packages listed below to pull in the relevant runtime.

Reply:
> In the context of ELN, there is no platform-python. We should probably exclude this one and just focus on the python3 packages listed below to pull in the relevant runtime.

Right, it makes sense, i'll update my patch.